sheets/views.py

```diff
@@ -125,13 +120,9 @@ def handle_watched_sheet_update(request):
         return HttpResponse(status=status.HTTP_400_BAD_REQUEST)
 
     sheet_type = request.GET.get("sheet", SHEET_TYPE_COUPON_REQUEST)
-    if sheet_type == SHEET_TYPE_COUPON_REQUEST:
-        sheet_metadata = CouponRequestSheetMetadata()
-    elif sheet_type == SHEET_TYPE_REFUND:
-        sheet_metadata = RefundRequestSheetMetadata()
-    elif sheet_type == SHEET_TYPE_DEFERRAL:
-        sheet_metadata = DeferralRequestSheetMetadata()
-    else:
+    try:
+        sheet_metadata = get_sheet_metadata_from_type(sheet_type)
+    except:
         log.error(
```

Comment:
Should this be `log.exception` or is the stack trace not important here?

Reply:
Correct. Stack trace isn't important